packages/compass-saved-aggregations-queries/src/actions/aggregations-queries-actions.ts

```diff
@@ -0,0 +1,9 @@
+import { AnyAction } from "redux";
+
+export enum actions {
+  FETCH_DATA = 'FETCH_DATA',
```

Comment:
suggestion: may be better to call the action like the function, ie. FETCH_DATA -> fetchData, FETCH_ITEMS -> fetchItems